### DISCUSSION TITLE
Problem about async_io is uncompatible

### DISCUSSION THREAD

**Discussion by niebowen666** [deepspeedai/DeepSpeed#6558] [Q&A] [answered]
I have installed the libaio1 and libaio-dev：

Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
libaio-dev is already the newest version (0.3.112-5).
libaio1 is already the newest version (0.3.112-5).
0 upgraded, 0 newly installed, 0 to remove and 23 not upgraded.

but I still get the warning：
[WARNING]  async_io requires the dev libaio .so object and headers but these were not found.
[WARNING]  If libaio is already installed (perhaps from source), try setting the CFLAGS and LDFLAGS environment variables to where it can be found.

Hope to get any answer！Thanks！

**Comment by tjruwase** [comment]
Can you please share the output of `ds_report`?

**Comment by loadams** [comment, marked as answer]
@niebowen666 - if this is Ubuntu24 you need to install `sudo apt install libaio1t64`

And you may need to link the libs after that: `sudo ln -s /usr/lib/x86_64-linux-gnu/libaio.so.1t64 /usr/lib/libaio.so.1`
